assets/locales/zh-cn/enhancements.ftl

```diff
@@ -0,0 +1,58 @@
+wine = Wine
+
+synchronization = 同步
+wine-sync-description = 同步 Wine 内部事件的技术
+
+language = 语言
+wine-lang-description = 设置 Wine 的语言。可以用于解决键盘布局问题
+system = 系统
+
+borderless-window = 无边框窗口
+virtual-desktop = 虚拟桌面
+
+game = 游戏
+
+hud = HUD
+
+fsr = FSR
+fsr-description = 将游戏分辨率提升到与屏幕相同。需要在游戏中选择较低分辨率，然后按 Alt+Enter
+ultra-quality = 超级画质
+quality = 画质优先
+balanced = 平衡画质与性能
+performance = 性能优先
+
+gamemode = 游戏模式
+gamemode-description = 提升游戏相对其他程序的优先级
+
+gamescope = Gamescope
+gamescope-description = Gamescope 是 Valve 开发的工具，可以让游戏运行在一个单独的 Xwayland 实例上。支持 AMD，英特尔和 Nvidia 的显卡
+
+discord-rpc = Discord RPC
+discord-rpc-description = Discord RPC 可以设置 Discord 状态，让你的好友知道你正在玩游戏
+title = 标题
+description = 描述
+
+fps-unlocker = 解除帧率限制
+
+enabled = 开启
+fps-unlocker-description = 修改游戏内存，解除渲染帧率限制。可能会触发反作弊检查
+
+power-saving = 省电模式
+power-saving-description = 当游戏失去焦点时 (例如切出游戏) 自动当帧率设为 10, 并降低进程优先级
+
+monitor = 显示器
+monitor-description = Number of monitor you want to run the game on
```

Comment:
Guess you missed this one?

Reply:
Updated. Was going to confirm if the "number" is the index or count. Verified that it should be the index.